sdk/inc/azure/core/internal/az_http_internal.h

```diff
@@ -222,21 +222,21 @@ AZ_NODISCARD az_result az_http_request_set_query_parameter(
  * @brief Add a new HTTP header for the request.
  *
  * @param ref_request HTTP request builder that holds the URL to set the query parameter to.
- * @param key Header name (e.g. `"Content-Type"`).
+ * @param name Header name (e.g. `"Content-Type"`).
```

Comment:
Nit, I prefer Key | Value over Name | Value

Reply:
In the case of pair, sure, but now this is more specific case, so I follow RFC 7230 terminology for HTTP message format (https://tools.ietf.org/html/rfc7230#section-3.2)